Lombiq.HelpfulLibraries/Libraries/Utilities/EnumerableExtensions.cs

```diff
@@ -15,7 +15,8 @@ public static class EnumerableExtensions
         /// <param name="asyncOperation">An <see langword="async"/> function to call on each item.</param>
         /// <typeparam name="TItem">The type of the input collection's items.</typeparam>
         /// <typeparam name="TResult">The type of the output collection's items.</typeparam>
-        /// <returns>When awaited the task contains the results which were added one-by-one.</returns>
+        /// <returns>When awaited, the <see cref="Task"/> that contains the results which were added one-by-one.</returns>
+

```

Comment:
```suggestion
```

Reply:
What do you mean? This one suggests removing line 19. Was that not your intention?

Reply:
It was. I was referring to my previous suggestion where the empty line was mistakenly added 😉